x-pack/plugins/security_solution/public/flyout/network_details/header.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React, { memo } from 'react';
+import { css } from '@emotion/react';
+import { EuiTitle } from '@elastic/eui';
+import type { EuiFlyoutHeader } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { FlowTargetSourceDest } from '../../../common/search_strategy';
+import { FlyoutHeader } from '../shared/components/flyout_header';
+
+const NETWORK_DETAILS = i18n.translate(
+  'xpack.securitySolution.timeline.sidePanel.networkDetails.title',
```

Comment:
consider updating to follow the expandable flyout format

Reply:
this was actually completely removed now that I'm using a similar UI as the user and host flyouts